src/ray/common/ray_config_def.h

```diff
@@ -84,7 +84,7 @@ RAY_CONFIG(uint64_t, raylet_check_gc_period_milliseconds, 100)
 /// memory_usage_threshold_fraction and free space is below the min_memory_free_bytes then
 /// it will start killing processes to free up the space.
 /// Ranging from [0, 1]
-RAY_CONFIG(float, memory_usage_threshold_fraction, 0.98)
+RAY_CONFIG(float, memory_usage_threshold_fraction, 0.95)
```

Comment:
why did we change this ti 0.95 from 0.98?

Reply:
To give more room chance to kill the process before it becomes worker crash, the original threshold for the legacy oom killer is also .95